components/infobox/wikis/rainbowsix/infobox_league_custom.lua

```diff
@@ -0,0 +1,273 @@
+	end
+	dateString = dateString:gsub('-XX', '-??')
+	return dateString
+end
+
+function CustomLeague:_createPrizepool()
+	if String.isEmpty(_args.prizepool) and String.isEmpty(_args.prizepoolusd) then
+		return nil
+	end
+	local date
+	if String.isNotEmpty(_args.currency_rate) then
+		date = _args.currency_date
+	end
+
+	return PrizePoolCurrency._get({
+		prizepool = _args.prizepool,
+		prizepoolusd = _args.prizepoolusd,
+		currency = _args.localcurrency,
+		rate = _args.currency_rate,
+		date = date or Variables.varDefault('tournament_enddate', _TODAY),
+	})
+end
+
+function CustomLeague:_createLiquipediaTierDisplay()
+	local tier = _args.liquipediatier or ''
+	local tierType = _args.liquipediatiertype or ''
+	if String.isEmpty(tier) then
+		return nil
+	end
+
+	local function buildTierString(tierString)
+		local tierText = Tier.text[tierString]
+		if not tierText then
+			table.insert(_league.warnings, tierString .. ' is not a known Liquipedia Tier/Tiertype')
+			return ''
+		else
+			return '[[' .. tierText .. ' Tournaments|' .. tierText .. ']][[Category:' .. tierText .. ' Tournaments]]'
+		end
+	end
+
+	local tierDisplay = buildTierString(tier)
+
+	if String.isNotEmpty(tierType) then
+		tierDisplay = buildTierString(tierType) .. '&nbsp;(' .. tierDisplay .. ')'
+	end
+
+	return tierDisplay
+end
+
+function CustomLeague:defineCustomPageVariables()
+	--Legacy vars
+	Variables.varDefine('tournament_ticker_name', _args.tickername or '')
+	Variables.varDefine('tournament_tier', _args.liquipediatier or '')
+	Variables.varDefine('tournament_tier_type', _args.liquipediatiertype or '')
+	Variables.varDefine('tournament_ticker_name', _args.tickername or '')
+	Variables.varDefine('tournament_prizepool', _args.prizepool or '')
+	Variables.varDefine('tournament_mode', _args.mode or '')
+	Variables.varDefine('tournament_currency', _args.currency or '')
+
+	--Legacy date vars
+	local sdate = Variables.varDefault('tournament_startdate', '')
+	local edate = Variables.varDefault('tournament_enddate', '')
+	Variables.varDefine('tournament_sdate', sdate)
+	Variables.varDefine('tournament_edate', edate)
+	Variables.varDefine('tournament_date', edate)
+	Variables.varDefine('date', edate)
+	Variables.varDefine('sdate', sdate)
+	Variables.varDefine('edate', edate)
+
+end
+
+function CustomLeague:_createGameCell(args)
+	if String.isEmpty(args.game) and String.isEmpty(args.patch) then
+		return nil
+	end
+
+	local content
+
+	local betaTag = String.isNotEmpty(args.beta) and 'Beta&nbsp;' or ''
+
+	if args.game == _GAME_SIEGE then
+		content = '[[Siege]][[Category:' .. betaTag .. 'Siege Competitions]]'
+	elseif args.game == _GAME_VEGAS2 then
+		content = '[[Vegas 2]][[Category:' .. betaTag .. 'Vegas 2 Competitions]]'
+	else
+		content = '[[Category:Tournaments without game version]]'
+	end
+
+	content = content .. betaTag
+
+	if String.isNotEmpty(args.epatch)  then
+		content = content .. '<br> [[' .. args.patch .. ']] &ndash; [[' .. args.epatch .. ']]'
```

Comment:
this assumes args.patch is set without checking if it is, could be a nil concat

Reply:
Removed it as it's unused on R6